tests/accounts/test_share.py

```diff
@@ -0,0 +1,111 @@
+#	THESE METHODS IN SHARE.PY ARE ALL TESTED THROUGH TEST_BUCKETS.PY
```

Comment:
The whole test_share.py is commented. If it is not required then delete it, else test it and commit it again.